packages/samples/common-types/openapi/v3/types.json

```diff
@@ -769,6 +775,7 @@
       "description": "The ID of the target subscription. The value must be an UUID.",
       "required": true,
       "type": "string",
+      "minLength": 1,
       "x-ms-parameter-location": "client"
```

Comment:
@live1206 we want to double check with you if adding this is ok (in relation with the original swagger) as this line `"x-ms-parameter-location": "client"` is not present in there: https://github.com/Azure/azure-rest-api-specs/blob/ad550d2f7adc2ca8f586080113eb220c3a6f3515/specification/common-types/resource-management/v3/types.json#L658

Reply:
I think default location is client.